React rework

railsstudent opened the pull request.

railsstudent committed:
Setup React and Material UI library

Initial commit

railsstudent committed:
Credit Component

railsstudent committed:
Add component to display hk kai font and default font of system

railsstudent committed:
Code cleanup

removed unused material-ui components
Group styles as objects of styles object
Add margin to buttons and rows in creditcardcomponent

railsstudent committed:
Delete unused css files


delete unused files

tonyhhyip commented:
Please upload some screen capture if possible

railsstudent commented:
Fixing it based on feedback

tonyhhyip commented:
@railsstudent Do you forget to git push?

railsstudent committed:
Make larger font size and increase line height

railsstudent commented:
![screen shot 2017-01-24 at 6 18 36 am](https://cloud.githubusercontent.com/assets/1159649/22254949/f38a00d0-e290-11e6-84af-c5e1b3502339.png)
![screen shot 2017-01-24 at 11 55 06 pm](https://cloud.githubusercontent.com/assets/1159649/22254956/f6395f60-e290-11e6-910a-7b1702022ad0.png)

railsstudent commented:
![screen shot 2017-01-24 at 11 54 57 pm](https://cloud.githubusercontent.com/assets/1159649/22255041/1ecf4a52-e291-11e6-88af-93f5cc30e150.png)

tonyhhyip commented on .yarnclean:
```diff
@@ -0,0 +1 @@
+
```
Please remove this file with `git rm --cached .yarnclean`

railsstudent replied:
Done

tonyhhyip commented on assets/js/InputCharacterComponent.jsx:
```diff
@@ -0,0 +1,75 @@
+    let numRows = 8;
+    return(
+      <div style={styles.container}>
+        <div style={styles.bigfontStyle}>
```
Font family of freehkfonts is missed

railsstudent replied:
Done

tonyhhyip commented on package.json:
```diff
@@ -0,0 +1,59 @@
+    "react-dom": "15.4.2",
+    "react-tap-event-plugin": "2.0.1"
+  },
+  "babel": {
```
You may remove the part as you have the .babelrc

railsstudent replied:
Done

tonyhhyip commented on public/index.html:
```diff
@@ -0,0 +1,34 @@
+
+    </script>
+
+    <!--script src="/js/app.js" type="text/javascript"></script -->
```
Please remove commented lines

railsstudent replied:
Done

tonyhhyip commented on assets/js/CreditComponent.jsx:
```diff
@@ -0,0 +1,80 @@
+
+class CreditComponent extends BaseMUI {
+
+  constructor(props) {
```
Remove this do nothing constructor

railsstudent replied:
Done

tonyhhyip reviewed (changes requested):
Great Job, just some minor issue and some way to be improved. Please take a look at the lines I comment on and consider to fix it

railsstudent committed:
1) Code cleanup
2) Create scss stylesheet and replace style object with style classname.

railsstudent commented:
Please review the changes again.

tonyhhyip commented:
Some place can improve like code style, I will handle it

railsstudent commented:
Sure, thanks.
